src/side-effects/biodiversity-data-manager/biodiversity-data-manager-component.jsx

```diff
@@ -0,0 +1,26 @@
+import { loadModules } from '@esri/react-arcgis';
+import { useEffect } from 'react';
+
+const RICHNESS_RARITY_GRID = 'rarity-richness-GRID';
+const TAXA_FIELD = 'TAXA';
+
+const BiodiversityDataManager = ({ map, setSpecies, setSpeciesLoading, setSpeciesError }) => {
```

Comment:
Maybe I'm kind of mistaken but this component looks like a `provider` to me. Wouldn't it make sense to it to sit on a `providers` folder. I mean, when I read `provider` I think of a component that feeds some data into the store.
Not sure if `side-effects` has a much broader meaning. What do you think?
Or is just me 🤷‍♂

Reply:
I was always thinking of providers as services that are fetching data asynchronously from the backend and in here this is not the case as we're taking data from a layer (no direct call to the API). That's my interpretation 🤷‍♀ We need a third pair of eyes maybe

Reply:
To be honest, I wrote only a few providers in my life so I'm not an authority here, but I took a look at how we solved providers in climate-watch and in my opinion it is similar to the provider's concept from CW. As Marta said, we are not fetching data asynchronously from the backend, we are passing `map` data as a prop, but also we are fetching some data `uniqueValues` through 'loadModules' from ArcGIS API. So to me, it looks more like provider, we are fetching part of the data in an async action and then we are passing the data to the store. Correct me if I'm wrong
![provideers](https://user-images.githubusercontent.com/15097138/58164845-80321480-7c7e-11e9-8a0d-7ed6fd6184ea.png)